ipaserver/plugins/idrange.py

```diff
@@ -764,4 +765,10 @@ def post_callback(self, ldap, dn, entry_attrs, *keys, **options):
         assert isinstance(dn, DN)
         self.obj.handle_ipabaserid(entry_attrs, options)
         self.obj.handle_iparangetype(entry_attrs, options)
+        self.add_message(
+            messages.ServiceRestartRequired(
+                service=services.service('sssd', api).systemd_name,
```

Comment:
Please use `services.knownservices['sssd'].systemd_name` instead -- you added 'sssd' to well-known services so there is no need anymore to instantiate a separate one, known services are already instantiated.